test/inline-compare/compare.py

```diff
@@ -67,6 +69,30 @@ def _python_packages(self):
 
         return packages, metadata
 
+    def _gem_packages(self):
+        packages = set()
+        metadata = collections.defaultdict(dict)
+        for entry in self._enumerate_section(
+                report="content-gem", section="content"
+        ):
+            package = Package(name=entry["package"], type=entry["type"].lower(),)
+            packages.add(package)
+            metadata[package.type][package] = Metadata(version=entry["version"])
+
+        return packages, metadata
+
+    def _npm_packages(self):
```

Comment:
Same.

Reply:
ditto, this'll make it in the npm package.json cataloger PR coming up